src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1909,15 +1920,18 @@ void PolicyHandler::OnSetCloudAppProperties(
 
   policy_manager_->InitCloudApp(policy_app_id);
 
+  bool auth_token_update = false;
   if (msg_params.keyExists(strings::enabled)) {
-    policy_manager_->SetCloudAppEnabled(policy_app_id,
-                                        msg_params[strings::enabled].asBool());
-
+    bool enabled = msg_params[strings::enabled].asBool();
+    policy_manager_->SetCloudAppEnabled(policy_app_id, enabled);
+    auth_token_update = enabled;
```

Comment:
Why should auth_token_update be set to enabled in this case?

Reply:
Updates are sent for all enabled apps when the policy table is loaded or updated. This logic is to cover the case where an app is enabled after the policy table is initially loaded, meaning that the listener does not already have the auth token for this app.